### ISSUE TITLE
Current windows version is unplayable slow as client

### ISSUE THREAD

**Issue by titiger** [MegaGlest/megaglest-source#86]
Current windows client works in single-player and as server without problems.
But if is a client the game is unplayable slow and choppy, This happens in Internet games and in local LAN games. I tested Linux as server( headlesss and non headless)  and windows as client.

Direct form the start you see that units don't move fluid. Just command some workers/fighters to walk across the screen to see the problem. This gets worse if there are more units in game, Evn simple mouse pointer movement is choppy. To me it looks like thread synchronisation trouble.
This problem is related to 32 and 64 bit windows client.

**Comment by tomreyn** [comment]
I can confirm this. Using Windows 10 64-bit, I tried several combinations:
* both 64- and 32-bit MG builds of 3.12-dev
* (32-bit) MG builds of 3.11.1
* local game, client to network game (hosted by Titi), network game host

All of these tests were carried out with network throttling disabled (i.e. the registry update was in place - I verified the registry setting using regedit, and there was no more warning on windwos throttling on MG's command line output).

On all games but the ones where I was on 3.12-dev as a network game client, UpdateFPS was around 120-140. Only in this case it kept switching back and forth between 39 and 55 constantly, with no action taking place on screen. RenderFPS was always at 40 in all tests / configurations.
